lib/recurly/hosted-fields.js

```diff
@@ -29,13 +30,55 @@ export class HostedFields extends Emitter {
     this.errors = [];
     this.initQueue = [];
     this.readyState = 0;
+    this.onTabNext = this.onTabNext.bind(this)
+    this.onTabPrevious = this.onTabPrevious.bind(this)
     this.configure(options);
     this.inject();
     this.on('hostedField:state:change', this.update.bind(this));
     this.on('bus:added', bus => {
       this.bus = bus;
       this.fields.forEach(hf => bus.add(hf));
     });
+
+    this.on('hostedField:tab:previous',this.onTabPrevious)
+    this.on('hostedField:tab:next',this.onTabNext)
```

Comment:
Since these handlers are only called on these events, let's bind them here.

```js
this.on('hostedField:tab:previous',this.onTabPrevious.bind(this);
```